tasks/31_reverse-string-in-range/reverse-string-in-range.js

```diff
@@ -0,0 +1,8 @@
+export function reverseStringInRange(str, index) {
+  const partOfText = str.slice(index[0], index[1] + 1);
+  const revertedPart = partOfText
+      .split('')
+      .reverse()
+      .join('');
+  return (str = str.replace(partOfText, revertedPart));
```

Comment:
this is fine solution, but do you have an idea how it could be done without those helper functions?